dask/array/ufunc.py

```diff
@@ -330,3 +330,9 @@ def modf(x):
     graph = HighLevelGraph.from_collections(right, rdsk, dependencies=[tmp])
     R = Array(graph, right, chunks=tmp.chunks, dtype=rdt)
     return L, R
+
+
+def divmod(x, y):
```

Comment:
You may want to use the `@copy_docstring` decorator, as is used on some of the other functions in this module.